src/providers/fault-store/fault-store.ts

```diff
@@ -66,4 +76,83 @@ export class FaultStoreProvider {
   getTotals() {
     return this.store.getState();
   }
+
+  getTestResult() {
+    let fs = 0;
+    let ds = 0;
+    let ss = 0;
+    const sectionRegEx = new RegExp(/^[a-z]*/);
+    const data = this.getTotals();
```

Comment:
Can we rename this function? It looks like it's getting the whole state, not totals.